webapp/src/Command/ImportEventFeedCommand.php

```diff
@@ -1006,7 +996,7 @@ protected function importGroup(array $event): void
         $this->eventLogService->log('groups', $category->getCategoryid(), $action,
                                     $this->contestId);
 
-        $this->processPendingEvents('group', $category->getCategoryid());
+        $this->processPendingEvents('group', $category->getExternalid());
```

Comment:
Can this be null?

Reply:
We assume you run in datasource>=1 mode when importing. We even check for this. So then it’s always set

Reply:
I'm thinking we should even block running this with datasource=0. Currently we still have the `allow-import-as-primary` flag but it can only break stuff, so why should we?

Reply:
What was the usecase for it?

Reply:
🤷‍♂️ I think it is never used. If we want to get rid of it I will do it as part of #1349